op-chain-ops/genesis/config.go

```diff
@@ -564,6 +578,8 @@ func (d *DeployConfig) RollupConfig(l1StartBlock *types.Block, l2GenesisBlockHas
 		EclipseTime:            d.EclipseTime(l1StartBlock.Time()),
 		FjordTime:              d.FjordTime(l1StartBlock.Time()),
 		InteropTime:            d.InteropTime(l1StartBlock.Time()),
+		// 4844 blobs usage activation for rollup DA
+		BlobsEnabledL1Timestamp: d.BlobsUpgradeTime(l1StartBlock.Time()),
```

Comment:
I believe that we should remove blobs enabled timestamp and use eclipse timestamp instead but should double check with @protolambda

Reply:
It's fine to leave it as-is with `BlobsEnabledL1Timestamp` since it's two hardforks out (`canyon` and `delta` first), but yes, ideally we prepare this for the `eclipse` upgrade proposal after.